uiplib/scheduler.py

```diff
@@ -2,79 +2,64 @@
 
 import os
 import time
-import sys
 from select import select
 import random
 
 from uiplib.utils.utils import get_percentage
-from uiplib.scrape import download
+from uiplib.scrape import get_image_data_list
+from uip;ib.scrape import download_and_store_image
```

Comment:
The code does not comply to PEP8.

*Origin: PEP8Bear, Section: `python`.*

The issue can be fixed by applying the following patch:

```diff
--- a/tmp/tmpb1p2gguw/uiplib/scheduler.py
+++ b/tmp/tmpb1p2gguw/uiplib/scheduler.py
@@ -7,7 +7,8 @@
 
 from uiplib.utils.utils import get_percentage
 from uiplib.scrape import get_image_data_list
-from uip;ib.scrape import download_and_store_image
+from uip
+ib.scrape import download_and_store_image
 
 try:
     import msvcrt
```